docs/assets/templates/services/template-big.dockstore.yml

```diff
@@ -18,6 +18,12 @@ service:
       role:
       affiliation:
 
+  # A short text description of the service, 140 characters or less in length.
+  # Service-wide setting that will affect ALL branches/tags; only set this as needed in a main branch.
+  # If no topic has been specified, Dockstore will use the repository's "About" description as the topic.
+  # Set this field to the empty string to reset the topic to the repository's "About" description.
+  topic:
+
```

Comment:
Should probably note upfront that this is optional (and not even necessarily recommended) a bit like the line below

Reply:
i.e. we recommend it more in the case of multiple workflows in a repo, not in general

Reply:
As I created this PR, I debated the following question with myself: should we prefer the user to specify a `topic` in `.dockstore.yml` for a single-entry repo?  In the end, I concluded "yes", for the following reasons;
1. If we can get the user to include a topic, for example by highlighting the `topic` field in the `.dockstore.yml` examples, we don't have to rely on them setting the repo "About" description, which is easy to forget or just not do.
2. The repo [and thus, its "About" description] may contain more than just stuff for Dockstore.
3. The user might want their repo "About" description to be longer than the topic limit.  I did a quick test, the github "About" description can be up to 351 (!?) chars long.

However, I can understand arguments in the other direction.  Would like to hear what others think...

Reply:
> If it's recommended, we should also note that 😁
> 
> I was mostly thinking in terms of the simplest case, a single workflow in the repo and them being too lazy to have a description in both the .dockstore.yml and the about field on GitHub. But I can be persuaded

This PR includes the `topic` field in most of the `.dockstore.yml` examples, and calls it out as "optional" in the long examples, so it feels like we're covering the two bases ("recommended" and "optional").  But if you don't think that's adequate, leave a comment and I'll try to figure something out.

Reply:
I do think the long example should say a few words about why we recommend it, partly due to the discussion above.

 i.e. it can be as simple at "you can add a description here if you don't have a description in the repo and especially if you have multiple workflows in a repo and want each one to have a unique description "

I think touching on the "why" is useful here

Reply:
> I do think the long example should say a few words about why we recommend it, partly due to the discussion above.
> 
> i.e. it can be as simple at "you can add a description here if you don't have a description in the repo and especially if you have multiple workflows in a repo and want each one to have a unique description "
> 
> I think touching on the "why" is useful here

I added a line to the comments in the "long" `.dockstore.yml` examples:
```
# Useful to specify unique topics for multiple notebooks in the same repository, or a topic that differs from the default.
```
I also reordered the lines to improve the flow.  Note that new line for the "service" example only includes the second part, because we don't support multiple services in the same repo.